### DISCUSSION TITLE
nova-2 not available when specifying tier

### DISCUSSION THREAD

**Discussion by robfig** [deepgram/community#441] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

nova-2 is now GA and I can access it by specifying model=nova-2

Our application has run-time flags for the model parameters and always passes a tier. I'm finding that neither `tier=nova` nor `tier=nova-2` are accepted when specifying `model=nova-2`. I suspect this is a minor oversight of the release, but it means that I'm not able to toggle it on without a code change to remove the tier parameter. 

```
https://api.deepgram.com/v1/listen?tier=nova&model=nova-2'
{"err_code":"INSUFFICIENT_PERMISSIONS","err_msg":"Project does not have access to the requested model." .. }

https://api.deepgram.com/v1/listen?tier=nova-2&model=nova-2'
{"err_code":"INVALID_QUERY_PARAMETER","err_msg":"Invalid 'tier' value of 'nova-2'." ... }

https://api.deepgram.com/v1/listen?model=nova-2'
{"err_code":"Bad Request","err_msg":"Bad Request: failed to process audio: corrupt or unsupported data" (success)
```

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?tier=nova-2&model=nova-2

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

3b9c0715-d9f2-4168-b745-9ccc0744c0f4

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @robfig, in order to immediately continue using the model/tier syntax, you can specify `model=2-general&tier=nova`. That will get you the Nova-2 general model.

However, we are moving toward encouraging users to only specify the `model` parameter (with no `tier` parameter) for ease of use, in which case you can reach the same model by specifying only `model=nova-2`. That's a forward-looking code change you can make when you're ready to modify your application.

**Reply by robfig** [reply]
Great! Will do, I need to upgrade the client I use anyway. Thanks for your help
